lib/graphiti/sideload.rb

```diff
@@ -326,6 +327,15 @@ def resource_class_loaded?
 
     private
 
+    def blank_query?(params)
+      if filter = params[:filter]
+        if filter.values == ['']
```

Comment:
This seems both too specific and too general?  What would happen in a scenario where filters = `{foo: '', bar:''}`? 

Should we instead be isolating to something like `params[:filter].has_key?(primary_key) && params[:filter][primary_key].empty?`?

Reply:
> Should we instead be isolating to something like `params[:filter].has_key?(primary_key) && params[:filter][primary_key].empty?`?

We can do this, but we actually catch less cases. That would fix the issue for specifically when graphiti is generating the params via PK/FK. The current implementation would *also* catch

```ruby
has_many :foos do
  params do |hash, model| # model is parent in graph
    hash[:filter] = { id: model.whatever }
  end
end
```

I really don't think we need to worry about all possible edge cases, but this seems reasonably likely to happen, and the current implementation works for this as well as graphiti-generated params.

Reply:
My main question in the second case would be whether handling this for the user is expected behavior or if understanding "blank" logic should be left to the user in this case.  I lean toward the former as well, but wanted to bring it up.  Fine with merging this if we're in agreement.

Reply:
We are 100% on the same page and that was my initial thought as well. End of the day, I think the above example is just too common to ignore - in fact, I was doing something like this originally when I first encountered the problem.